### ISSUE TITLE
Implement a new highlight as a separate layer, like the DevTools one.

### ISSUE THREAD

**Issue by captainbrosset** [captainbrosset/devtools-highlighter#3]
Implement a new highlight as a separate layer, like the DevTools one.

**Comment by captainbrosset** [comment]
Of course, if there was a WebExtension API to just use the built-in DevTools highlighter, that would be awesome too.
@rpl do you know if this is something that's on the roadmap for Firefox extensions?

**Comment by rpl** [comment]
@captainbrosset it is not in the roadmap, but it would be an interesting devtools API to prototype 
(e.g. to get a better picture about how it could be exposed as a WebExtensions API).

**Comment by rajatvijay** [comment]
@captainbrosset do you think prototyping an API like the built in devtools highlighter is a good approach to solve this issue? or should we figure out a somthing simpler pertaining just the use case?
